src/components/SignUp/OffstageRoles.tsx

```diff
@@ -1,22 +1,263 @@
+        value: 'Special Effect Design'
+      },
+      {
+        label: 'Master Electrician',
+        value: 'Master Electrician'
+      }
+    ]
+  },
+  sound: {
+    textHeader: 'Sound',
+    checkboxes: [
+      {
+        label: 'Sound Design',
+        value: 'Sound Design'
+      },
+      {
+        label: 'Sound Mixer / Engineer',
+        value: 'Sound Mixer / Engineer'
+      }
+    ]
+  },
+  hairMakeupCostumes: {
+    textHeader: 'Hair, Makeup, Costumes',
+    checkboxes: [
+      {
+        label: 'Costume Design',
+        value: 'Costume Design'
+      },
+      {
+        label: 'Hair & Wig Design',
+        value: 'Hair & Wig Design'
+      },
+      {
+        label: 'Make-up Design',
+        value: 'Make-up Design'
+      }
+    ]
+  }
+};
 
 const OffstageRoles: React.FC<{
   setForm: any;
   formData: any;
 }> = props => {
+  const { formData, setForm } = props;
+  const {
+    offstageRolesGeneral,
+    offstageRolesProduction,
+    offstageRolesScenic,
+    offstageRolesLighting,
+    offstageRolesSound,
+    offstageRolesHairMakeupCostumes
+  } = formData;
+
+
+  const isRoleInRolesSection = (section: string[], role: string) =>
+  section.indexOf(role) > -1;
+
+ 
+  const rolesSectionChange = (chkValue: any, chk: string, fieldName: string, section: string[]) => {
+    let sectionRoles = [...section];
+    let name = fieldName;
+
+    if (chkValue) {
+      if (sectionRoles.indexOf(chk) < 0) {
+        sectionRoles.push(chk);
+      }
+    } else {
+      sectionRoles = sectionRoles.filter(rG => rG !== chk);
+    }
+
+    const target = {
+      name: 'offstageRolesGeneral',
+      value: sectionRoles
+    };
+
+    setForm({ target });
+  };  
+  
   return (
-    <Container>
-      <Row>
-        <Col lg="12">
-          <h1>So, What Do You Like Doing?</h1>
-          <h2>Tell us what positions suit you best</h2>
-        </Col>
-      </Row>
-    </Container>
+    <div>
+      <Title>SO, WHAT DO YOU LIKE DOING?</Title>
+      <TitleSecond>Tell us what positions suit you best.</TitleSecond>
+      <TitleThird>Off-Stage Roles</TitleThird>
+      <TitleFourth>Select all applicable positions</TitleFourth>
+
+      {Object.keys(offstageRolesObj).map(objKey => {
+        const currObjKey = (offstageRolesObj as any)[objKey as any];
+
+        return (
+          <>
+            <Col lg="10">
+              <GenericAccordion
+                key={`accordion-${currObjKey.textHeader}`}
+                textHeader={currObjKey.textHeader}
+              >
+                <Container>
+                  <Row>
+                    <Col lg="6">
+                      {currObjKey.checkboxes.map(
+                        (chk: { label: any; value: any }) => {
+                          const { label: chkLabel, value: chkValue } = chk;
+
+                          return (
+                            <Checkbox
+                              checked={isRoleInRolesSection(offstageRolesGeneral, chk)}
```

Comment:
The next step for our functionality changes this to:
```
checked={isRoleInRolesSection(sectionStateValue, chk)}
```
Because our section is going to be different depending on which instance we're on in our map, so we need to grab it dynamically from the object